### PR TITLE
get_library_items method VerificationLibrary 

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This PR addresses the development of the ```get_library_items``` method in the ```VerificationLibrary``` class.

### CODE REVIEW

File: src/api/verification_library.py [hunk 1]
```diff
@@ -100,3 +100,25 @@ def get_library_item(self, item_name: str) -> Dict:
...
+            Dict that contains summary information of library items.
+        """
+
+        # check `items` arg type
+        if not isinstance(items, List):
+            logging.error(f"items' type must be List. It can't be {type(items)}.")
+            return None
+
+        # get each lib item's summary info
+        item_list = []
```
Comment (leijerry888) [inline src/api/verification_library.py:131]: By default, get all library items loaded at instantiation. this logic needs to be added here. please also add a test for this.

File: src/api/verification_library.py [hunk 2]
```diff
@@ -100,3 +100,25 @@ def get_library_item(self, item_name: str) -> Dict:
...
         return item_dict
+
+    def get_library_items(self, items: List = []) -> Union[List, None]:
+        """Get the json definition and meta information of a list of specific library items.
+
+        Args:
+            items:  list of str, default []. Library items to get. By default, get all library items loaded at instantiation.
+
+        Returns:
+            Dict that contains summary information of library items.
```
Comment (leijerry888) [inline src/api/verification_library.py]: list of dict. not dict

**Review by leijerry888 — CHANGES_REQUESTED** [review]
Nice work. Please see my inline comments

File: src/api/verification_library.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply src/api/verification_library.py]: Addressed.

File: src/api/verification_library.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply src/api/verification_library.py:131]: I'm not sure if I understood this point. Could you be more specific? 

From my understanding, the goal of the ```get_library_items``` method is to grab multiple libraries' info. And we can iterate the ```get_library_item``` method to get this job done.

Comment (yunjoonjung-PNNL) [reply src/api/verification_library.py:131]: Addressed.

**Review by lymereJ — APPROVED** [review]

**Review by leijerry888 — APPROVED** [review]

**Pull request merged**